### ISSUE TITLE
No error is shown for large file uploads that fail

### ISSUE THREAD

**Issue by alexclifton4** [kanboard/kanboard#5171]
### Checklist

- [x ] I verified that Kanboard is correctly installed
- [x ] I verified that the problem does not come from a plugin
- [x ] I verified that the problem is not already reported
- [x ] I understand that Kanboard is in maintenance mode. It doesn't mean it's abandoned, but there is no significant feature development

### Actual behaviour

When adding an attachment to a task, if the file is larger than the server's maximum upload size, the server will return HTTP 413 (Content too large), but Kanboard doesn't catch this and the uploading spinner will remain indefinitely.

### Expected behaviour

Kanboard should notice the HTTP 413 error code and show an error to the user.

### Steps to reproduce

- Take note of the server's maximum file upload size, i.e. `upload_max_filesize` in php.ini
- Try to attach a document to a task that is larger than this size.
- Observe that it will appear to be stuck uploading, but the HTTP 413 error will appear in the developer console.

### Configuration

- Kanboard version: 1.2.25
- Database type and version: 
- PHP version: postgres 14.7 (Ubuntu 14.7-0ubuntu0.22.04.1)
- OS: Linux 5.15.0-1031-aws
- Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36

**Comment by Ruel-Strude** [comment]
You sould also change `post_max_size`. @see http://php.net/post-max-size

**Comment by alexclifton4** [comment]
Sorry for any confusion, I know how to configure PHP / nginx / etc. for this to work correctly.

My report was more pointing out that Kanboard does not handle the error correctly if the server is not configured this way.
There are legitimate reasons why an administrator may want a smaller max upload size

**Comment by alexclifton4** [comment]
I've just upgraded to v1.2.27 and the error is still there.
I'm using nginx with php-fpm.
As a test, I set the `client_max_body_size` directive in `nginx.conf` to 1M then tried to upload an image larger than this, the screenshot shows what happens in this case: The 'Upload files' button spins forever, while the console shows that the server responded with a 413.
![Screenshot from 2023-03-27 13-52-29](https://user-images.githubusercontent.com/20780073/227945101-ddca5e4e-a612-4174-a414-f9f3ae923356.png)

**Comment by alexclifton4** [comment]
Yes, I was imagining that the fix for this would be on the client side - this way the user could receive an error for any upload failures, not just this specific case.
